system/include/emscripten/wasmfs.h

```diff
@@ -57,7 +74,9 @@ backend_t wasmfs_create_memory_backend(void);
 //
 // TODO: Add an async version of this function that will work on the main
 // thread.
-backend_t wasmfs_create_fetch_backend(const char* base_url __attribute__((nonnull)));
+//
+backend_t wasmfs_create_fetch_backend(const char* base_url __attribute__((nonnull)),
+                                      uint32_t chunk_size);
```

Comment:
Please mention this API change in `ChangeLog.md`

Reply:
Will do.